caffe2/CMakeLists.txt

```diff
@@ -1068,15 +1068,14 @@ if(USE_XPU)
   if(NOT TARGET torch_xpu_ops)
     message(WARNING "Failed to include ATen XPU implementation target")
   else()
-    target_link_libraries(torch_xpu PRIVATE torch_xpu_ops)
     if(MSVC)
       # Windows
       target_link_libraries(torch_xpu PRIVATE
       "-WHOLEARCHIVE:\"$<TARGET_FILE:torch_xpu_ops>\"")
     else()
       # Linux
       target_link_libraries(torch_xpu PRIVATE
-        "-Wl,--whole-archive,\"$<TARGET_FILE:torch_xpu_ops>\" -Wl,--no-whole-archive")
+        "-Wl,--whole-archive" torch_xpu_ops "-Wl,--no-whole-archive")
```

Comment:
One more question, have you evaluated the impact?

Reply:
Sorry, I don't get the `impact` means ? We can pass the CI.

Reply:
`\"$<TARGET_FILE:torch_xpu_ops>\" ` should be the full path, while `torch_xpu_ops` is not. Any impact?